testColorPair.cpp

```diff
@@ -0,0 +1,33 @@
+#include "testColorPair.h"
+#include <assert.h>
+
+void testColorPair::displayPairMap()

```

Comment:
New functionality is bundled together with test code. Keep test code and functionality separate.